schema.sql

```diff
@@ -8,5 +8,6 @@ CREATE TABLE animals(
     date_of_birth date NOT NULL,
     escape_attempts int NOT NULL,
     neutered boolean NOT NULL,
-    weight_kg decimal NOT NULL
+    weight_kg decimal NOT NULL,
+    species text NOT NULL
```

Comment:
- [ ] It will be great to alter the table instead of adding `species` directly to your already created table. This is because its safe and won't cause any problems to your already established table. To do this you should do something like
`ALTER TABLE animals ADD species varchar(255)`. This should be added out of your  CREATE TABLE statement. 
Check the example bellow:
```
CREATE TABLE animals(
    date_of_birth date NOT NULL,
    escape_attempts int NOT NULL,
    neutered boolean NOT NULL,
    weight_kg decimal NOT NULL
);

ALTER TABLE animals ADD species varchar(255)

```

Reply:
Hello @RolandM99. I did that in the command line and I think I forgot that to add in the vscode